src/App/Types.hs

```diff
@@ -56,6 +57,20 @@ data ProjectMetadata = ProjectMetadata
 instance ToJSON ProjectMetadata where
   toEncoding = genericToEncoding defaultOptions
 
+data ProjectMetadataRevision = ProjectMetadataRevision
```

Comment:
I'm not sure what the significance of `Revision` in this name is. For each of it's fields I think the `Revision` text at the end of the name isn't necessary.

Reply:
Agreed but I had to name it something different due to this existing data obj:

```
data ProjectMetadata = ProjectMetadata
  { projectTitle :: Maybe Text
  , projectUrl :: Maybe Text
  , projectJiraKey :: Maybe Text
  , projectLink :: Maybe Text
  , projectTeam :: Maybe Text
  , projectPolicy :: Maybe Policy
  , projectLabel :: [Text]
  , projectReleaseGroup :: Maybe ReleaseGroupMetadata
  }
  deriving (Eq, Ord, Show, Generic)
```

Reply:
It sounds like you made a new type that would be specific to the edit command. Is that right? Perhaps `ProjectMetadataEditSubcommand`?